features/RST15_audit_events_admin_ui.feature

```diff
@@ -0,0 +1,95 @@
+Feature: Audit events convey useful information via admin interface
+
+    Background:
+	
+	Given fixtures from "bdd_auth" are loaded
+	And fixtures from "bdd_auth" are available
+        And fixtures from "auditevent_valid_excercise_ui" are loaded
```

Comment:
Could we wrap this into one step?
What does this background actually do? We should discuss this.